spinta/manifests/tabular/helpers.py

```diff
@@ -1446,6 +1457,22 @@ def _resource_to_tabular(
     yield from _lang_to_tabular(resource.lang)
 
 
+def _base_to_tabular(
+    base: Base,
+    *,
+    end_of_base: bool = False
+) -> Iterator[ManifestRow]:
+    if end_of_base:
+        yield torow(DATASET, {
+            'base': '/',
+        })
```

Comment:
This is not related to base, since any dimension can have end marker. So it would be more correct to move this to a separate function `_end_marker(name)`, which would produce end marker for any given dimension by name.